safety.sh

```diff
@@ -0,0 +1,18 @@
+#/bin/bash
+currenttime=`date +"%d-%b-%Y %H:%M:%S"`
+logfile=`date +"%d-%b-%Y"`_log.log
+echo "================= $currenttime"  >> $logfile
+RMLEDGER=1500
+CLUSTERSLOT=$(solana slot -u http://10.142.0.4:8899)
+NODESLOT=$(solana slot -u http://127.0.0.1:8899)
+DIFFSLOT=$(($CLUSTERSLOT-$NODESLOT))
+echo "Cluster Slot:" $CLUSTERSLOT "Current Slot:" $NODESLOT "Difference in Slots:" $DIFFSLOT >> $logfile
+if [[ $DIFFSLOT -gt $RMLEDGER ]]; then
+        cd ~
+        ./stop
+        rm -rf ledger/
+        ./restart
+        echo "Node was:" $DIFFSLOT " slots behind, Services has been stopped, ledger deleted and service restarted" >> $logfile
```

Comment:
Should we emit a metric here so we can get flagged in pager-duty and or on http://bit.ly/solana-cluster-sanity?

Emiting a metric is easy, start by sourcing this script: https://github.com/solana-labs/cluster/blob/19a50f11e00403d13b9ef7520afce17ea2fc811d/bin/warehouse.sh#L31